### ISSUE TITLE
Symbols-1000-em Nerd Font Complete.ttf has broken letter 'E'

### ISSUE THREAD

**Issue by fwsmit** [ryanoasis/nerd-fonts#581]
#### 🎯 Subject of the issue

The capital letter 'E' from 'Symbols-1000-em Nerd Font Complete.ttf' is not working correctly. It seems like either it only contains that letter, or that letter is prioritized over other fonts. Anyhow, this leads to the letter 'E' not showing up in mono fonts in firefox. See below for an example.

Arch bug report: https://bugs.archlinux.org/task/69568?project=5&string=ttf-nerd-fonts-symbols-mono
Forum post: https://bbs.archlinux.org/viewtopic.php?pid=1955896

#### 🔧 Your Setup

- 'Symbols-1000-em Nerd Font Complete.ttf'
- Program: firefox
- OS: Arch linux.
- Package: ttf-nerd-fonts-symbols-mono 2.1.0+36+gd0bf73a1-2

#### ★ Screenshot

![example](https://user-images.githubusercontent.com/23078054/106174382-44f72100-6195-11eb-912d-0a5670384213.png)

**Comment by StaticRocket** [comment]
bumping the issues because it's still prevalent
![external-content duckduckgo com](https://user-images.githubusercontent.com/35777938/114814079-6d002480-9d81-11eb-8aaa-4d1f6d313c39.png)

**Comment by polyzen** [comment]
In the meantime you could use a more tailored font configuration file, eg.:
https://github.com/polyzen/dotfiles/blob/master/workstation/.config/fontconfig/fonts.conf

I haven't used 10-nerd-font-symbols.conf and don't have this issue.

**Comment by Strahinja** [comment]
It also seems as if capital "E" is represented by blank space only in "1000 em" version and not in "2048 em" version (previews by [fontpreview](https://github.com/sdushantha/fontpreview)):
![https://i.imgur.com/tPxgY6b.png](https://i.imgur.com/tPxgY6b.png)
![https://i.imgur.com/okEx2TR.png](https://i.imgur.com/okEx2TR.png)
![https://i.imgur.com/pMoP0Pj.png](https://i.imgur.com/pMoP0Pj.png)
![https://i.imgur.com/8vn62eo.png](https://i.imgur.com/8vn62eo.png)

**Comment by Strahinja** [comment]
I think I have a workaround. I am using Artix Linux (which is just Arch with non-systemd init), and previously I had manually installed both "1000 em" and "2048 em" versions of the font. After removing all the manually installed files and running:
```
$ pacman -S ttf-nerd-fonts-symbols
```
I can see the capital letter "E" in the example:
![https://i.imgur.com/wT6u5sp.png](https://i.imgur.com/wT6u5sp.png)
Of course, the issue still persists in the "1000 em" version of the font, which is included in the package `ttf-nerd-fonts-symbols-mono`.

**Comment by lilydjwg** [comment]
I managed to use fontconfig to exclude `E` to the point that

```sh
fc-list "Symbols Nerd Font":charset=0x45
```

shows empty. But Firefox still uses the blank `E` from this font. :shrug:

**Comment by WhyNotHugo** [comment]
I can't quite figure out how the `Symbols-1000-em Nerd Font Complete.ttf` is built. The contributing instructions mostly cover updating or adding other fonts, but not that particular one. Can't find any script that generates it or alike.

Any pointers here?

**Comment by Finii** [comment]
They are created by
```
fontforge font-patcher 'src/glyphs/NerdFontsSymbols 1000 EM Nerd Font Complete Blank.sfd' --complete --ext ttf
```
and as far as I can see they contain no `E`.

Well, there are two variants of that font in the repo, unfortunately:
* https://github.com/ryanoasis/nerd-fonts/blob/master/src/glyphs/Symbols-1000-em%20Nerd%20Font%20Complete.ttf
* https://github.com/ryanoasis/nerd-fonts/blob/master/patched-fonts/NerdFontsSymbolsOnly/complete/Symbols-1000-em%20Nerd%20Font%20Complete.ttf

The one in `src/` is obsolete. It's not clear to me why it is still there, maybe 'keep old links valid' reasons.

**Comment by Finii** [comment]
Obviously Ryan does not patch from the `sfd` ...
Creating PR.

**Comment by WhyNotHugo** [comment]
> and as far as I can see they contain no E.

![image](https://user-images.githubusercontent.com/730811/156573035-e49d060c-c787-4571-a97f-0ba3398a8233.png)

It took me a while to figure out what you meant :joy: 

> The one in src/ is obsolete. It's not clear to me why it is still there, maybe 'keep old links valid' reasons.

The README clearly indicates that one should not use master or point to master. I think it's sensible to drop one to avoid any confusion.

>  Creating PR.

Many thanks, I clearly have little idea on this one.

**Comment by Finii** [comment]
Regarding 'keep paths', compare this https://github.com/ryanoasis/nerd-fonts/issues/495#issuecomment-1030712701

**Comment by Finii** [comment]
Fixed with #801
